### ISSUE TITLE
List mode: Game result wrongly positioned when last move is black

### ISSUE THREAD

**Issue by braoult** [mliebelt/pgn-viewer#414]
There is a display issue for result (`1-0`, `0-1`, etc...) in `list` mode when last move is black :

![image](https://user-images.githubusercontent.com/8958429/214785382-ea2139fb-99f9-4397-a70a-3ee0657ea4b1.png)

When last move is white, the display is correct :

![image](https://user-images.githubusercontent.com/8958429/214785744-f75b70cd-72e4-42e7-af2c-7cba8de2365f.png)

Maybe an option would be to always display the result below the last move line ?

**Comment by mliebelt** [comment]
Agree. It should be a simple fix to change the styling to something like a comment, or even have a separate CSS class or selector for it. I will have a look into it.
